web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -98,6 +98,35 @@
           }
           return false;
         };
+        
+        /**
+         * @description
+         * Returns a Layer already added to the map.
+         *
+         * @param {ol.Map} map obj
+         * @param {string} name of the layer
+         * @param {string} url of the service
+         */
+        var getTheLayerFromMap = function(map, name, url) {
```

Comment:
Just to be sure the changes in this file are related to the improvements of Feature Catalogue?

Reply:
Yes, the changes to this part are for improving the visualization on the map (decoding the information shown in the table can be improved with associated feature catalog content)

Reply:
Hi, sorry for dropping by unexpectedly, do you think you could refactor the `isLayerInMap` method just above to use this code? i.e. doing `return getTheLayerFromMap(map, name, url) != null`

Lately we've been trying to reduce the amount of code in this service by removing/deprecating duplicate code. Thank you!

Reply:
Right! Done. Thanks for dropping by! :)